src/pages/docs/guides/chat/build-embedded-chat.mdx

```diff
@@ -0,0 +1,414 @@
+- Do you need any spam prevention measures applied before allowing users to join rooms?
+  - For example, in a support chat scenario, use a captcha or other measures before allowing users to join a room or create a new ticket.
+- Do you need long-term archiving of the chat history?
+  - Check out the guide for [exporting chat data](/docs/guides/chat/export-chat) for an overview of your options.
+
+Ably has been tested to handle millions of concurrent connections and rooms. Whether you're managing 100 support conversations or 100,000 simultaneous delivery chats, the architecture remains the same - no re-engineering required as you grow.
+
+## Authentication
+
+Authentication is a design decision. 
+
+In some cases, you'll want to tie participants to their identity in your application. In others, anyone can join and participate. Ably Chat supports both of these scenarios - you don't need to go through the hassle of setting up users in Ably and trying to keep them in sync with your application. Every client has a `clientId`. If you want to allow random users, just use a random string. If you want to tie it to your application, then use the ID of the user or some other well-known identifier. Just make sure you can tell them apart if you allow both at the same time, for example by using distinct prefixes.
+
+<Code>
+```javascript
+const jwt = require("jsonwebtoken");
+
+const header = {
+    "typ": "JWT",
+    "alg": "HS256",
+    "kid": "{{ API_KEY_NAME }}"
+}
+const currentTime = Math.round(Date.now() / 1000);
+
+// The clientId could be the users ID in your application,
+// or a random string if you want to make users anonymous.
+//
+// The capabilities here allow the holder of this token to publish and subscribe to messages,
+// typing indicators and room reactions in the chat room called "foo".
+
+const claims = {
+    "iat": currentTime, /* current time in seconds */
+    "exp": currentTime + 14400, /* time of expiration in seconds */
+    "x-ably-capability": "{\"foo\":[\"publish\", \"subscribe\"]}",
+    "x-ably-clientId": "your-client-id",
+}
+
+ablyJwt = jwt.sign(
+    claims,
+    "{{ API_KEY_SECRET }}",
+    { header: header }
+)
+
+console.log('JWT is: ' + ablyJwt);
+```
+</Code>
+
+How you authenticate is also key. To balance security and experience, you want short-lived tokens that can be easily revoked if a user is misbehaving or needs their permissions changed, but automatically expire after a period of time. This means that if a token is compromised, it will only be valid for a limited time. **In production apps, you should not use API keys for client-side authentication**. You can use them server-side, but as they are long-lived and require explicit revocation, exposure to untrusted users poses a continuing risk.
```

Comment:
I don't think we necessarily need the point about API keys here considering the examples and text are all focussed on JWTs.